main.rb

```diff
@@ -59,3 +59,14 @@
 puts "Date: #{rental.date}"
 puts "Person: #{rental.person.correct_name}"
 puts "Book: #{rental.book.title}"
+
+# Create an instance of LibraryApp
+library_app = LibraryApp.new
+
+# Use the LibraryApp methods
+library_app.list_all_books
+library_app.list_all_people
+library_app.create_person(25, 'Emma', false)
+library_app.create_book('Ruby Programming', 'John Doe')
+library_app.create_rental(Date.new(2023, 12, 20), 1, 'The Great Gatsby')
+library_app.list_rentals_for_person(1)
```

Comment:
- [ ]  Great work on making the console so far 👍 but I guess you missed the requirements because you are supposed to make a console app a user can interact with means a user can insert books, persons, and rentals and get them by interacting with your console but when I tried to run the console app then it showed me already inserted inputs and their outputs and this is not what is the requirement 😨 Kindly make your app interactive for the user to follow the project requirements 😊